metricbeat/module/memcached/stats/stats.go

```diff
@@ -11,9 +11,9 @@ import (
 )
 
 func init() {
-	if err := mb.Registry.AddMetricSet("memcached", "stats", New); err != nil {
-		panic(err)
-	}
+	mb.Registry.MustAddMetricSet("memcached", "stats", New,
+		mb.DefaultMetricSet(),
```

Comment:
No host parser here?

Reply:
memcache was one of the early modules before we had the host parser feature. We should probably add one but would not make it part of this PR.

Reply:
Ok